### PR TITLE
Add platform integrations metrics origins

### PR DESCRIPTION
**Pull request by sarah-witt**
### What does this PR do?
Adding a batch of platform integrations origins

### CODE REVIEW

File: releasenotes/notes/add-platform-integrations-metrics-origins-8fbb73d9d9614c72.yaml [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+other:
+  - |
+    Add metric origins for Platform Integrations Fly.io, Kepler, Octopus Deploy, and Scaphandre.
```
Comment (drichards-87) [inline releasenotes/notes/add-platform-integrations-metrics-origins-8fbb73d9d9614c72.yaml:11]:
```suggestion
    Add metric origins for Platform Integrations: Fly.io, Kepler, Octopus Deploy, and Scaphandre.
```

**Review by drichards-87 — APPROVED** [review]
Left a tiny suggestion from Docs and approved the PR.

**Review by jszwedko — APPROVED** [review]
Approving @DataDog/agent-processing-and-routing owned files.

**Review by carlosroman — APPROVED** [review]
Added `qa/done` flag.

**Review by iliakur — APPROVED** [review]

**Pull request merged**